cloud-sql/mysql/servlet/src/main/java/com/example/cloudsql/ConnectorConnectionPoolFactory.java

```diff
@@ -73,6 +73,12 @@ public static DataSource createConnectionPool() {
     config.addDataSourceProperty("ipTypes", "PUBLIC,PRIVATE");
     // [START cloud_sql_mysql_servlet_connect_unix]
 
+    // Setting cloudSqlRefreshStrategy to lazy
+    // to refresh the tokens when they are needed, rather than on a scheduled interval.
```

Comment:
"refresh the tokens" is vague language and can cause confusion. Let's avoid this kind of language in our docs.

A "refresh" refreshes not only an OAuth2 token but also the ephemeral certificate.

```java
// cloudSqlRefreshStrategy set to "lazy" is used to perform a
// refresh when needed, rather than on a scheduled interval.
// This is recommended for serverless environments to
// avoid background refreshes from throttling CPU.
```